### DISCUSSION TITLE
The 'visual' field in frontmatter

### DISCUSSION THREAD

**Discussion by EpicMandM** [groepl/Obsidian-Zettelkasten-Starter-Kit#6] [Q&A] [answered]
What is the purpose of the 'visual' field in frontmatter? Can you provide guidance on its usage and application?

**Comment by groepl** [comment, marked as answer]
It's used for Dataview queries, when the visual should be shown in a table or list. But I think it could be nice to have some examples added in the template repository. Stay tuned, I'll share some on GitHub.

**Reply by EpicMandM** [reply]
Thank you, groepl! I'm looking forward to the examples you'll share. Your contributions to the community are greatly appreciated!
